frontends/verilog/Makefile.inc

```diff
@@ -6,7 +6,7 @@ GENFILES += frontends/verilog/verilog_lexer.cc
 
 frontends/verilog/verilog_parser.tab.cc: frontends/verilog/verilog_parser.y
 	$(Q) mkdir -p $(dir $@)
-	$(P) $(BISON) -Wall -Werror -o $@ -d -r all -b frontends/verilog/verilog_parser $<
+	$(P) $(BISON) --debug  -Wall -Werror -o $@ -d -r all -b frontends/verilog/verilog_parser $<
```

Comment:
Was this left in by accident?

Reply:
Yeah, oops, I'll fix that, one moment

Reply:
Fixed, rebased and push -f

Please review again